example/src/App.js

```diff
@@ -90,6 +91,13 @@ function App() {
       ReactGA.pageview(location.pathname + location.search);
     }
   }, [location]);
+  useEffect(() => {
+    setAppType(isRestMode ? "Editor" : "Viewer");
+  }, [isRestMode]);
+  useEffect(() => {
+    document.title = `OSCAL ${appType}`;
+  }, [appType]);
+
```

Comment:
```suggestion
useEffect(() => {
    setAppType(isRestMode ? "Editor" : "Viewer");
    document.title = `OSCAL ${appType}`
}, [isRestMode]);
```
Using `useEffect` twice seems unnecessary. Why not do this instead?

Reply:
@mikeisen1 so apparently this isn't quite cool with React. See the message on commit [a2d084f](https://github.com/EasyDynamics/oscal-react-library/pull/398/commits/a2d084fc71579ea000721774b704ad41c53ab6f3).